### PR TITLE
Web-Design-Match —> Web-Design

### PR DESCRIPTION
**Pull request by marianbitsov23**
Stable web version.
Added design on the home page.
Major bug fixes.

### CODE REVIEW

**Review by msimov02 — APPROVED** [review]
LGTM!

File: Web/tues_pairs/src/components/Pages/Home/home.jsx [hunk 1]
```diff
@@ -27,12 +19,14 @@ class UserList extends Component {
...
       users: [],
+      currentUser: null,
     };
   }
 
   onMatch = event => {
-    const currentUser = this.props.authUser;
+    const currentUser = this.state.currentUser;
 
+    console.log("Here");
```
Comment (GGLozanov) [inline Web/tues_pairs/src/components/Pages/Home/home.jsx]: Изтрии това.

**Review by GGLozanov — CHANGES_REQUESTED** [review]
Малка промяна за debug print-а, иначе LGTM.

**Review by GGLozanov — APPROVED** [review]
Добре, LGTM.

**Pull request merged**